simexpal/schemes/experiments.json

```diff
@@ -40,21 +40,40 @@
 		},
 		"slurm_settings": {
 			"type": "object",
-			"properties": {
-				"num_nodes": {
-					"type": ["integer", "string"],
-					"minimum": 1
-				},
-				"procs_per_node": {
-					"type":  ["integer", "string"],
-					"minimum": 1
-				},
-				"num_threads": {
-					"type": ["integer", "string"],
-					"minimum": 1
+			"allOf": [
+				{
+					"not": {
+						"anyOf": [
+							{
+								"required": ["exclusive", "num_threads"],
```

Comment:
We do not want to limit the user to either use `exclusive` or `num_threads`. Both can be set.